src/deployment/registration.py

```diff
@@ -516,6 +516,35 @@ def assign_scaleset_role(onefuzz_instance_name: str, scaleset_name: str) -> None
         assign_scaleset_role_manually(onefuzz_instance_name, scaleset_name)
 
 
+def set_app_audience(objectId: str, audience: str) -> None:
```

Comment:
is this being used ?

Reply:
Good call out. Turns out I was missing code from the previous PR. I'll push out the changes in a little bit.